os2mo_data_import/os2mo_data_import/mora_data_types.py

```diff
@@ -785,6 +785,7 @@ def build(self):
 
         if self.seperate_names:
             self.payload = {
+                "user_key": self.user_key,
                 "givenname": self.givenname,
                 "surname": self.surname,
                 "cpr_no": self.cpr_no,
```

Comment:
Rather than setting the entire payload for each conditional, only the key/value pairs for givenname, surname and name should be updated on the existing payload.